doc/time-series.rst

```diff
@@ -223,16 +224,26 @@ Through the standalone ``cftime`` library and a custom subclass of
 functionality enabled through the standard :py:class:`pandas.DatetimeIndex` for
 dates from non-standard calendars or dates using a standard calendar, but
```

Comment:
Can we clarify "non-standard calendars" -> "non-standard calendars commonly used in climate science"

This paragraph is missing a bit of context for non-climate users :)